docs/source/get-involved/gsoc/projects_2025/brainglobe.md

```diff
@@ -1,14 +1,14 @@
 # GSoC NIU Projects 2025: BrainGlobe
 
+[BrainGlobe](https://brainglobe.info) is a community-driven suite of open-source Python tools. The BrainGlobe tools are widely used to process, analyse and visualise images of brains (and other related data) in neuroscientific research.
 If you are interested in any of these projects, [get in touch](https://brainglobe.info/contact.html)! Feel free to open a new topic on [Zulip](https://brainglobe.zulipchat.com/) and tag the potential mentors.
 
 Our working language is English, but our mentors for these projects also speak Italian, French, and German.
 
-:::{dropdown} {fas}`video;sd-text-primary` Improve `cellfinder`'s classification algorithm
+:::{dropdown} {fas}`brain;sd-text-primary` Improve `cellfinder`'s classification algorithm
 <!-- Description -->
-BrainGlobe is a community-driven suite of open-source Python tools. The BrainGlobe tools are widely used to process, analyse and visualise images of brains (and other related data) in neuroscientific research.
 
-The BrainGlobe `cellfinder` tool is used to detect cells in large whole-brain images. It uses signal processing to find possible cell candidates and passes them to a customisable classifier to split the candidates into cells and no-cells. `cellfinder` relies heavily on `pytorch` and `keras`.
+The [BrainGlobe `cellfinder` tool](https://brainglobe.info/documentation/cellfinder/index.html) is used to detect cells in large whole-brain images. It uses signal processing to find possible cell candidates and passes them to a customisable classifier to split the candidates into cells and no-cells. `cellfinder` relies heavily on `pytorch` and `keras`.
 
 `cellfinder` currently uses a residual neural network (ResNet) to classify cell candidates, and Deep Learning network architectures have progressed since. In this project, you will explore newer architectures for the classification network as alternatives to ResNet, and see whether they work better and/or faster than the existing implementation.
```

Comment:
Not part of this PR, but I don't think DL needs to be capitalised. 

```suggestion
`cellfinder` currently uses a residual neural network (ResNet) to classify cell candidates, and deep dearning network architectures have progressed since. In this project, you will explore newer architectures for the classification network as alternatives to ResNet, and see whether they work better and/or faster than the existing implementation.
```